redactions/widget.js

```diff
@@ -124,18 +125,9 @@ export default function (state = initialState, action) {
  * @param {string} datasetId
  * @returns {Promise<void>}
  */
-function fetchDataset(datasetId) {
-  return (dispatch, getState) => {
-    const state = getState();
-    const datasetService = new DatasetService(datasetId, {
-      apiURL: process.env.WRI_API_URL,
-      language: state.common.locale
-    });
-
-    return datasetService.fetchData('metadata')
-      .then(dataset => dispatch({ type: SET_WIDGET_DATASET, payload: dataset }));
-  };
-}
+const fetchDataset = datasetId =>
```

Comment:
For consistency, this should be `getDataset`. We reserve the `fetch-` for functions that perform requests to services. And you solve the naming issue with `fetchDatasetService ` ;)